### ISSUE TITLE
Vocabulary

### ISSUE THREAD

**Issue by 211938554** [udaysankar01/xfeatSLAM#7]
hello,
    Thank you for your work.
    I've noticed that GCNV2 and Superpoint  , they all have a corresponding Vocabulary , but yours doesn't. Is that correct?

**Comment by AlbertoRemus** [comment]
> Hello, that's correct. Right now, I am using the same Vocabulary as ORB-SLAM3.

Hi @udaysankar01, but if use Xfeat, is it problematic to employ ORB vocabulary?

**Comment by udaysankar01** [comment]
> > Hello, that's correct. Right now, I am using the same Vocabulary as ORB-SLAM3.
> 
> Hi [@udaysankar01](https://github.com/udaysankar01), but if use Xfeat, is it problematic to employ ORB vocabulary?

Yes, directly using ORB vocabulary on XFeat features will be problematic. I have got to train a new vocabulary specific to XFeat features to be able to perform relocalization directly with XFeat features.

**Comment by AlbertoRemus** [comment]
@udaysankar01 thanks for your reply!
